docs/backend-api/how-to/how to get track points.md

```diff
@@ -0,0 +1,75 @@
+How to get track points for trips
+
+Sometimes necessary to get all points of a trip with more info about the device's moves.
+How to get them?
+
+Firstly you need to [get hash](http://not-doing.ru/tmp/mkdocs/backend-api/how-to/get-session-hash/)
+
+Once you get the hash, you need to [get your tracker_id](http://not-doing.ru/tmp/mkdocs/backend-api/how-to/get-tracker-list/). The platform must know points for what device must be in reply.
+
+Now you can get all points for the interesting period using track/read API call:
+
+    https://api.navixy.com/api-v2/track/read?hash=your_hash&tracker_id=device's_id&from=yyyy-MM-dd HH:mm:ss&to=yyyy-MM-dd HH:mm:ss
+
+Parameters that necessary for this call:
+
+* Tracker_id - we got them in tracker/list call. Use only one tracker_id per call. It should be an integer
+
+* From - A string containing start date/time in "yyyy-MM-dd HH:mm:ss" format (in user's timezone)

```

Comment:
References to special data types must be provided with a link to `../getting-started.md#data-types`.